controller/store/sql/postgresql/017_v0_4_23_password_reset_request_unique.sql

```diff
@@ -0,0 +1,7 @@
+-- +migrate Up
+
+-- remove the old unique index (users might need multiple password resets)
+ALTER TABLE password_reset_requests DROP CONSTRAINT password_reset_requests_account_id_key;
+
+-- add new constraint which doesnt mind having multiple resets for account ids
+ALTER TABLE password_reset_requests ADD CONSTRAINT password_reset_requests_account_id_key FOREIGN KEY (account_id) REFERENCES accounts (id) on delete cascade;
```

Comment:
why `on delete cascade`? also... let's standardize on SQL keywords being in `CAPS`... so if that's needed, let's do `ON DELETE CASCADE`.

Reply:
Updated case. I was trying to keep on top of potential dangling data, doesn't seem necessary to me to keep around reset requests for completely deleted accounts

Reply:
There's a `deleted` flag on this table. All of the tables are set up for soft deletes. It provides an audit trail for a lot of the data.